projects/Allwinner/patches/linux/0100-tanix-mmc-rtw88-keep-power-in-suspend.patch

```diff
@@ -0,0 +1,10 @@
+--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-tanix.dtsi	2022-06-25 14:21:11.340219073 +0000
++++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-tanix.dtsi	2022-06-26 09:52:05.018133375 +0000
+@@ -168,6 +168,7 @@
+ 	mmc-pwrseq = <&wifi_pwrseq>;
+ 	bus-width = <4>;
+ 	non-removable;
++	keep-power-in-suspend;
+ 	status = "okay";
```

Comment:
This patch should be merged with 0054-arm64-dts-allwinner-h6-tanix-tx6-enable-rtw88-wifi-in-dts.patch

Reply:
I’m not ready to merge this, because it hasn’t fixed my standby issue. I still have to unload the driver. Haven’t done much on this for a while. I did notice this today in the lkml - https://lore.kernel.org/lkml/822ed6b6-fd3a-3873-cdf0-543340fd4103@leemhuis.info/

Reply:
Closing as this is in the rtw88 PR